browser/ui/webui/settings/brave_default_extensions_handler.h

```diff
@@ -23,15 +25,23 @@ class BraveDefaultExtensionsHandler : public settings::SettingsPageUIHandler {
  private:
   // SettingsPageUIHandler overrides:
   void RegisterMessages() override;
-  void OnJavascriptAllowed() override;
-  void OnJavascriptDisallowed() override;
+  void OnJavascriptAllowed() override {}
+  void OnJavascriptDisallowed() override {}
 
   void GetRestartNeeded(const base::ListValue* args);
   void SetWebTorrentEnabled(const base::ListValue* args);
   void SetHangoutsEnabled(const base::ListValue* args);
   void SetIPFSCompanionEnabled(const base::ListValue* args);
   void SetMediaRouterEnabled(const base::ListValue* args);
   void SetBraveWalletEnabled(const base::ListValue* args);
+#if BUILDFLAG(ENABLE_TOR)
+  void SetTorDisabled(const base::ListValue* args);
```

Comment:
please use `Enabled`, not `Disabled`. It's easier to understand and more consistent with everything else.

Reply:
Agreed that negative expression makes difficult to read/understand.
How about this leave as a f/u?
Just using `enabled` in this class also could increase complexity because related pref name is `kTorDisabled` and options name is also `Disable Tor`. Also some related static methods are `IsTorDisabled()` and already used by many other places.
I think it would be good to clean this up as a separate PR. WDYT?

Reply:
@bridiver Changed to `Enabled`.